### DISCUSSION TITLE
About issue deadline

### DISCUSSION THREAD

**Discussion by sapatevaibhav** [VaibhavCodeClub/learn#33] [Announcements]
I will adhere to the guidelines set forth by GSSoC'24, which stipulate a deadline of 5 days for each contributor. Should you become inactive on the assigned issue, it will be reassigned to the next contributor. If you encounter extenuating circumstances such as exams or other commitments, please communicate this by adding a comment related to your situation.

**Comment by hariram4862** [comment]
Sure
